moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -115,8 +114,8 @@ void CollisionCheck::run()
 
   collision_result_.clear();
   // Self-collisions and scene collisions are checked separately so different thresholds can be used
-  getLockedPlanningSceneRO()->getCollisionEnvUnpadded()->checkSelfCollision(collision_request_, collision_result_,
-                                                                            *current_state_, acm_);
+  getLockedPlanningSceneRO()->getCollisionEnvUnpadded()->checkSelfCollision(
+      collision_request_, collision_result_, *current_state_, getLockedPlanningSceneRO()->getAllowedCollisionMatrix());
```

Comment:
I see that `getAllowedCollisionMatrix()` returns a reference to the ACM so I think there shouldn't be any hit to performance  :+1: